flow/projecttracking/tracker/tracker.py

```diff
@@ -76,34 +96,38 @@ def _retrieve_story_detail(self, story_id):
         method = '_retrieve_story_detail'
         commons.print_msg(Tracker.clazz, method, 'begin')
 
-        tracker_story_details_url = Tracker.tracker_url + '/services/v5/projects/' + Tracker.project_id + '/stories/' + story_id
-
-        headers = {'Content-type': 'application/json', 'Accept': 'application/json', 'X-TrackerToken': Tracker.token}
+        tracker_story_details = []
+        json_data = None
+        resp = None
 
-        commons.print_msg(Tracker.clazz, method, tracker_story_details_url)
+        if Tracker.project_ids is not None:
+            for project_id in Tracker.project_ids:
+                tracker_story_details.append(
+                    {'url': Tracker.tracker_url + '/services/v5/projects/' + project_id + '/stories/' + story_id})
```

Comment:
Not sure if there are any gotchas, but we can probably use the following API to get around having to loop through all the `project_id`s

```
export TOKEN='your Pivotal Tracker API token'

curl -X GET -H "X-TrackerToken: $TOKEN" "https://www.pivotaltracker.com/services/v5/stories/559"
```

From: https://www.pivotaltracker.com/help/api/rest/v5#Stories (Does not let me link directly to the specific endpoint, scroll down through this section).